src/vellum/workflows/workflows/base.py

```diff
@@ -84,7 +83,7 @@
 class _BaseWorkflowMeta(type):
     def __new__(mcs, name: str, bases: Tuple[Type, ...], dct: Dict[str, Any]) -> Any:
         if "graph" not in dct:
-            dct["graph"] = []
+            dct["graph"] = set()
```

Comment:
fixes a bug with serializing a base workflow with no graph attribute defined, noticed during the happy path push test. Added a serializer test to repro